contributors/devel/pull-requests.md

```diff
@@ -142,6 +143,16 @@ Either the [on call contributor](on-call-rotations.md) will manage the merge que
 
 That's the last step. Your PR is now merged.
 
+## Marking Unfinished Pull Requests
+
+If you want to solicit reviews before the implementation of your pull request is complete, you should hold your pull request to ensure that the merge queue does not pick it up and attempt to merge it. There are two methods to achieve this:
+
+1. You may use the `/hold` and `/hold cancel` comment commands
+2. You may prefix your pull request title with `WIP` or `[WIP]`
+
+The GitHub robots will add and remove the `do-not-merge/hold` label as you use the comment commands and the `do-not-merge/work-in-progress` label as you edit your title. While either label is present, your pull request will not be considered for merging.
```

Comment:
How to remove a `do-not-merge/work-in-progress` label? Can it be removed by removing the `WIP` prefix in the PR title?